apps/fulfillment-e2e/src/integration/start-picking-picklist-with-customer-note.cy.ts

```diff
@@ -0,0 +1,122 @@
+import { CustomerNoteFragment } from '../support/page_fragments/customer-note.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const customerNoteFragment = new CustomerNoteFragment();
+
+describe('Start picking a picklist with customer note', () => {
+  let identifier = '';
+
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.visit('/');
+    pickingListsFragment
+      .getCustomerNoteButton()
+      .parents('oryx-picking-list-item')
+      .within(() => {
+        pickingListsFragment
+          .getIdentifier(true)
+          .invoke('text')
+          .then((text) => {
+            identifier = text.trim();
+
+            pickingListsFragment.getStartPickingButton(true).click();
+          });
+      });
+  });
+
+  it('should navigate to correct url', () => {
+    cy.url().should('include', `/customer-note-info/${identifier}`);
+  });
+
+  it('should display customer note', () => {
+    customerNoteFragment
+      .getNote()
+      .should('be.visible')
+      .and('contain', 'Some cart note');
+  });
+
+  it('should display “Customer note” headline', () => {
+    customerNoteFragment.getHeadline().should('contain', 'Customer note');
+  });
+
+  it('should display illustration', () => {
+    customerNoteFragment
+      .getIllustration()
+      .should('be.visible')
+      .and('have.attr', 'resource', 'user-note');
+
+    customerNoteFragment
+      .getIllustration()
+      .invoke('outerWidth')
+      .should('be.eq', 250);
+
+    customerNoteFragment
+      .getIllustration()
+      .invoke('outerHeight')
+      .should('be.eq', 250);
```

Comment:
I'm not sure that CSS checks are needed in e2e (these will be covered by UI tests which we will add later)